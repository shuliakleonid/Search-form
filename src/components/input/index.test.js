import React from 'react';
import { render } from '@testing-library/react';
import Input from './input';

describe('Input component', ()=>{
  let sut;
  let props;

  it('should match snapshot', ()=>{
    sut = render(<Input {...props}/>)

    expect(sut).toMatchSnapshot();
  })
})
