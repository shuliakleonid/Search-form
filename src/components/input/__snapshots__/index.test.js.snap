// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should match snapshot 1`] = `undefined`;

exports[`Input component with props should match snapshot 1`] = `
<input
  className="input-field test"
  name="search"
  placeholder="Input you data"
/>
`;

exports[`Input component without props should match snapshot 1`] = `
<input
  className="input-field test"
/>
`;

exports[`Input component without props should match snapshot 2`] = `
<input
  className="input-field test"
/>
`;
